index.html

```diff
@@ -194,7 +226,6 @@ <h4>FEC website</h4>
       </div>
       <div class="tablet:grid-col-2"></div>
     </div>
-  </div>
```

Comment:
I don't think we want to remove this. If you remove  a closing tag like this one, it can disrupt the layout of the page